api/v1beta1/glanceapi_types.go

```diff
@@ -69,7 +69,8 @@ type GlanceAPISpec struct {
 	Secret string `json:"secret"`
 
 	// +kubebuilder:validation:Optional
```

Comment:
do we support setup glanceapi without glance ? if not, we could also set this to be required on the glanceapi and set via the glance controller when create the glanceapi?